main.go

```diff
@@ -0,0 +1,167 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/appcenter"
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-steputils/tools"
+	"github.com/bitrise-io/go-utils/log"
+)
+
+const statusEnvKey = "APPCENTER_DEPLOY_STATUS"
+
+type config struct {
+	Debug              bool            `env:"debug,required"`
+	ApkPath            string          `env:"apk_path,file"`
+	AppName            string          `env:"app_name,required"`
+	APIToken           stepconf.Secret `env:"api_token,required"`
+	OwnerName          string          `env:"owner_name,required"`
+	Mandatory          bool            `env:"mandatory,required"`
+	MappingPath        string          `env:"mapping_path"`
+	ReleaseNotes       string          `env:"release_notes"`
+	NotifyTesters      bool            `env:"notify_testers,required"`
+	DistributionGroup  string          `env:"distribution_group"`
+	DistributionStore  string          `env:"distribution_store"`
```

Comment:
Can we add parsing and additional validation to these freeform input params, before doing anything else? DistributionStore supposedly has predefined valid values.

Reply:
Not sure about this. @lszucs ?

Reply:
There is no predefined enum for that in the API docs: https://openapi.appcenter.ms/#/ I think it is not a good approach to define them by ourselves. I would leave this for now.

Reply:
Very good comment, I myself tried to find the predefined values, but I did not find any.

We could definitely spend some time fooling around with the API in a next iteration to collect the values and improve the step.

Reply:
Where does the end user know what to input here in this case?

Reply:
`appcenter distribute stores <command>` may list these